plugins/node/opentelemetry-plugin-ioredis/src/types.ts

```diff
@@ -17,7 +17,7 @@
 import * as ioredisTypes from 'ioredis';
 import { PluginConfig } from '@opentelemetry/api';
 
-export interface IORedisCommand {
+export interface IoRedisCommand {
```

Comment:
```suggestion
export interface IoredisCommand {
```

and all occurences of `IoRedis` as ioredis is one word https://github.com/luin/ioredis